scripts/fix-schemas-with-refs.ts

```diff
@@ -0,0 +1,49 @@
+import path from "path";
+import fsPromises from "fs/promises";
+import { omit, pick } from "lodash";
+
+const fixSchemasWithRefs = (object: any) => {
+  if (Array.isArray(object)) {
+    return object.map((value) => fixSchemasWithRefs(value));
+  }
+  if (object instanceof Object) {
+    const keys = Object.keys(object);
+    if (keys.includes("oneOf") && object.oneOf instanceof Object) {
+      return {
+        ...object,
+        oneOf: object.oneOf.map((oneOf: any) => {
+          if (!(oneOf instanceof Object) || !oneOf?.["$ref"]) {
+            return oneOf;
+          }
+          return pick(oneOf, "$ref");
```

Comment:
Tu nie boli nas to, że np. usuwane są description z obiektu?

Reply:
nie boli to i tak nigdy to description sie nie wyświetli